pom.xml

```diff
@@ -216,6 +222,12 @@
 		</dependency>
 
 		<dependency>
+			<groupId>org.apache.httpcomponents</groupId>
```

Comment:
So `aws-java-sdk` has an undeclared dependency to HC? Strange ...

I'll check all that this weekend. Thanks a lot for your work !